examples/look/look/images.py

```diff
@@ -1,28 +1,23 @@
 import io
+import json
 import mimetypes
 import os
+import re
 import uuid
 
-import msgpack
-
 import falcon
 
 
-class Resource:
+class Collection:
     def __init__(self, image_store):
         self._image_store = image_store
 
     def on_get(self, req, resp):
-        doc = {
-            'images': [
-                {
-                    'href': '/images/1eaf6ef1-7f2d-4ecc-a8d5-6e8adba7cc0e.png',
-                },
-            ],
-        }
-
-        resp.data = msgpack.packb(doc, use_bin_type=True)
-        resp.content_type = 'application/msgpack'
+        max_size = req.get_param_as_int('maxsize', min_value=1, default=-1)
+        images = self._image_store.list(max_size)
+        doc = {'images': [{'href': '/images/' + image} for image in images]}
+
+        resp.text = json.dumps(doc, ensure_ascii=False)
```

Comment:
let's use media here. What do you think @vytas7 ?

Reply:
I think it may be a good idea, but in that case we might need to revise the whole tutorial, and either convert to media wholesale (I'm not against it), or illustrate both approaches, but then we need to explain to the reader why we choose one or another.

Reply:
talked with @vytas7 and we already had in program to use media in the tutorial, but that would be another issue.

so it's fine as is for now!